### PR TITLE
Support arm64 builds

### LINKED ISSUE #57: Create arm64 image for secretgen-controller
**Describe the problem/challenge you have**
Currently, secretgen-controller cannot run on an ARM64 host, as there is no image[0].

Could we switch to doing multi-arch builds and produce arm64 alongside amd64?

Related issue for kapp-controller: https://github.com/vmware-tanzu/carvel-kapp-controller/issues/573

This is needed for Tanzu Community Edition

### PR DESCRIPTION
**Pull request by neil-hickey**
Mostly borrowed from https://github.com/vmware-tanzu/carvel-kapp-controller/pull/680

Fixes https://github.com/vmware-tanzu/carvel-secretgen-controller/issues/57

Signed-off-by: Neil Hickey <nhickey@vmware.com>

### CODE REVIEW

File: config/dev/values-schema.yml [hunk 1]
```diff
@@ -12,3 +12,5 @@ dev:
   image_repo: docker.io/k14s/secretgen-controller-test
   #@schema/desc "Development version"
   version: develop
+  #@schema/desc "Comma separated list of supported architectures"
+  platform: ""
```
Comment (cppforlife) [inline config/dev/values-schema.yml]: @praveenrewar is coming after you :)

Comment (benmoss) [reply config/dev/values-schema.yml]: you should configure your editor to always put a newline at eof

**Review by benmoss — APPROVED** [review]
lgtm other than newline nits

File: config/dev/values-schema.yml (hunk 1 above)
Comment (praveenrewar) [reply config/dev/values-schema.yml]: 😂

Comment (praveenrewar) [reply config/dev/values-schema.yml]: > you should configure your editor to always put a newline at eof

+1

Comment (neil-hickey) [reply config/dev/values-schema.yml]: > you should configure your editor to always put a newline at eof

yeh I moved from VSCode --> VSCode 2 and I keep forgetting to turn that setting back on. FINE. Guilted into it lol

**Pull request merged**